### DISCUSSION TITLE
i have a problem on my pc

### DISCUSSION THREAD

**Discussion by stavealexe** [community/community#121182] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

my all file and application convert to exe file. new downloded file also covert to exe file . what I do now.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by fdistasio** [comment]
Hello, this might be useful to you:
https://answers.microsoft.com/en-us/windows/forum/all/all-my-files-turned-into-exe/d9a29ca5-dfbf-40c9-ac3f-11e6a9ce6488

**Comment by davevad93** [comment]
Hi @stavealexe , If I remember correctly, to solve this problem you have to download a fix using another computer (because if you download it directly it will be saved as .exe file) and save it on a USB or external drive, then connect it to your computer and run the fix.
In any case check this Microsoft Support Community thread, download the `exefix.reg`  and follow all the steps thoroughly: 
- [All my folders files got converted into exe](https://answers.microsoft.com/en-us/windows/forum/windows_11-wintop_backup/canceled/5fe77660-62ec-4c70-94ee-6d378e7f9b7c?messageId=e9ea80ba-4828-4d0b-b802-acfc6a733c10)

**Reply by stavealexe** [reply]
It's not work

**Comment by niladri-1** [comment, marked as answer]
Hi, This web site help You,

https://www.quora.com/All-my-folders-are-turning-into-a-exe-application-by-hiding-original-folders-in-my-laptop-running-on-Windows-10-How-can-I-fix-this

The ".exe virus" is a common threat spread by portable devices, which hides actual folders and creates fake executable files to spread further. These viruses can disable crucial system functions like Folder options, Task Manager, and Regedit. To address this:

Scan your USB device and computer for viruses.
Use Command Prompt (CMD) to recover infected files:
Open CMD as administrator.
Enter commands:
cd\
(Change drive if necessary)
Dir/ah
Attrib *. -h -s /s /d
Manually remove the virus using Command Prompt:
Enter commands to kill virus processes:
taskkill /f /t /im "New Folder.exe"
taskkill /f /t /im "SCVVHSOT.exe"
taskkill /f /t /im "SCVHSOT.exe"
taskkill /f /t /im "scvhosts.exe"
taskkill /f /t /im "hinhem.scr"

**Reply by stavealexe** [reply]
thanks
